test/test.js

```diff
@@ -1,16 +1,81 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import Enzyme from 'enzyme';
+import { shallow, mount, configure } from 'enzyme';
```

Comment:
Lines 2 and 3 are redundant. Line 2 imports the entire `Enzyme` object. You can just use Enzyme.shallow, Enzyme.mount, and Enzyme.configure. Alternatively, remove line 2 and just use shallow, mount, and configure.